src/services/window.ts

```diff
@@ -46,6 +46,17 @@ ipcMain.on('create-extra-window', (event, arg) => {
 });
 
 async function createMainWindow(app: App) {
+    app.setUserTasks([
+        {
+            program: process.execPath,
+            arguments: '--new-window',
+            iconPath: process.execPath,
+            iconIndex: 0,
+            title: 'Open New Window',
+            description: 'Open new window'
```

Comment:
I think it's also worth to set localization with i18n here.

Reply:
@JYC333 , that would be nice but unfortunately we don't have server-side internationalization support yet. I hope we'll have it sooner than later but I wouldn't block the PR for that.

Reply:
I see, I think we have it all the place.